### PR TITLE
SSB4: Bugfixes and Balance changes

### PR DESCRIPTION
**Pull request by xInstruct**

### CODE REVIEW

**Comment by xInstruct** [comment]
finally managed to push through my headache and test.

Ready to be reviewed and/or merged

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -4259,8 +4189,8 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		accuracy: true,
 		basePower: 0,
 		category: "Status",
-		desc: "Raises the Special Attack of the user, and either Defense or Special Defense randomly, by 1 stage.",
-		shortDesc: "Gives +1 SpA and +1 Def or SpD",
+		desc: "Raises the Defense and Special Defense by 1 stage. Lowers the foe's higher offensive stat by 1 stage.",
+		shortDesc: "+1 Def & SpD. -1 to foe's highest offensive stat.",
```
Comment (AnnikaCodes) [inline data/mods/ssb/moves.ts:4193]:
```suggestion
		shortDesc: "Raises user's Def and SpD by 1. -1 to foe's highest offensive stat.",
```

Comment (xInstruct) [reply data/mods/ssb/moves.ts:4193]: Too long, gets cut when using /dt

Comment (xInstruct) [reply data/mods/ssb/moves.ts:4193]: image

**Comment by xInstruct** [comment]
Would like this merged before the start of april please :3

I have other SSB stuff that needs to be added in a seperate PR soon, so if this gets merged i can pull the commit to my other branch and finish working on it.

File: data/mods/ssb/random-teams.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ export const ssbSets: SSBSets = {
 	},
 	Cake: {
 		species: 'Dunsparce', ability: 'Wonder Guard', item: 'Shell Bell', gender: 'M',
-		moves: ['Haze', 'Life Dew', ['Poison Gas', 'Corrosive Gas', 'Magic Powder', 'Speed Swap', 'Spite', 'Refresh', 'Screech', 'Trick Room', 'Heal Block', 'Geomancy']],
+		moves: ['Haze', 'Jungle healing', ['Poison Gas', 'Corrosive Gas', 'Magic Powder', 'Speed Swap', 'Spite', 'Refresh', 'Screech', 'Trick Room', 'Heal Block', 'Geomancy']],
```
Comment (AnnikaCodes) [inline data/mods/ssb/random-teams.ts:185]:
```suggestion
		moves: ['Haze', 'Jungle Healing', ['Poison Gas', 'Corrosive Gas', 'Magic Powder', 'Speed Swap', 'Spite', 'Refresh', 'Screech', 'Trick Room', 'Heal Block', 'Geomancy']],
```

**Comment by AnnikaCodes** [comment]
@HoeenCoder?

**Review by HoeenCoder — COMMENTED** [review]
Looks good, thanks!

**Pull request merged**